modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -514,6 +517,66 @@ public void testBindPublishedObjectDefinitions() throws Exception {
 					_treeFactory.createObjectDefinitionTree(
 						objectDefinition1.getObjectDefinitionId()),
 					_objectDefinitionLocalService));
+
+		// Bind two published object definitions trees
```

Comment:
Bind two published object definitions that belong to a hierarchical structure

Reply:
Done!